src/main/java/com/example/SOPSbackend/controller/AdminController.java

```diff
@@ -19,8 +31,68 @@ public ResponseEntity<String> hello() {
         return ResponseEntity.ok("hello");
     }
 
+    @GetMapping("doctors")
+    public ResponseEntity<Object> showDoctors(@RequestParam Optional<Integer> page) {
```

Comment:
Page chyba powinien być obowiązkowy
I dodałbym autoryzację (see: DoctorController, praktycznie każdy mapping)

Reply:
Wydaje mi się, że 

![image](https://user-images.githubusercontent.com/52869032/163511237-ac386507-90bc-4dbe-ba89-9c1f93c14ea9.jpeg)

domyślnie bierzemy pierwszą stronę

Reply:
Co do autoryzacji to masz na myśli to coś czy coś innego?: ![image](https://user-images.githubusercontent.com/52869032/163511852-fae10219-d869-4d27-a870-e5569f999886.jpeg)

Reply:
@V1OL3TF0X

Reply:
Chodzi o @Secured({Role = ....}) chyba

Reply:
Dodałem w złym pliku. Teraz już dodane.